Базовая стилизация всех страниц

OlgaNikl opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2067969/tasks/6)

OlgaNikl committed:
исправлены лого и звезды

заменила лого на svg; разметила
в div

OlgaNikl committed:
Добавляет базовую стилизацию

Подключены шрифты; добавлены базовые текстовые параметры; подключены основные декоративные изображения

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,291 @@
+  font-size: 18px;
+  line-height: 21px;
+  color: #000000;
+  background-color: #ffffff;
```
фон у body серый 
![HTML 1 : Седона 2022-06-02 22-59-24](https://user-images.githubusercontent.com/11165551/171736420-70962b5d-006c-4d9d-bc22-19d527321f4e.jpg)
белый - это фоновый цвет контейнера, в котором лежит весь контент страницы. то есть надо добавить обёртку вокруг хедера, мэйн и футера
https://jsfiddle.net/theano/dcmqy1r2/33/ вот примерно как здесь добавлен див с классом container

OlgaNikl replied:
эти ссылки открываются только с vpn

talequale replied:
сорри! не знала. сервисы vpn уже недоступны?

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,291 @@
+}
+
+.user-list-button {
+  background: #756157;
```
лучше не использовать короткое свойство background, только развёрнутые. то есть тут надо вместо background писать background-color. дело в том, что когда мы пишем `background: #756157;`, мы задаём не только фоновый цвет блоку, но и одновременно перезаписываем вообще все фоновые характеристики, которые у него могут быть 
это видно, если в инструментах разработчика нажать на треугольничек 
![HTML Academy: Седона 2022-06-03 00-13-28](https://user-images.githubusercontent.com/11165551/171739411-f015b07e-ae10-462e-8fdf-9c0b8a74841d.jpg)

background-color задаётся значение, которое мы указали, а все остальные свойства получают значение  initial. это может быть проблемой, когда элемент попадает под действие нескольких селекторов

talequale replied:
помимо фона надо ещё text-transform: uppercase указать. но это верно для всех кнопок

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,291 @@
+  color: #ffffff;
+}
+
+.area-item,
+.tourists-item {
+  background: rgba(131, 179, 211, 0.12);
+}
+
+.road-item {
+  background: rgba(131, 179, 211, 0.2);
+}
```
тут вообще не такая логика, фон блокам раздаётся не по классу, а по порядковому номеру. блоки просто красятся через один, нечётные в светло-голубой, чётные в более тёмный голубой
как вот тут https://jsfiddle.net/theano/dcmqy1r2/33/ 
![Edit fiddle - JSFiddle - Code Playground 2022-06-03 00-19-43](https://user-images.githubusercontent.com/11165551/171740303-08c57de5-bd95-4bfb-bd07-9b3dbcd6fed3.jpg)

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,291 @@
+.housing.town-advantages-item,
+.souvenirstown-advantages-item {
+  background: rgba(131, 179, 211, 0.12);
+}
```
в этом списке такая же история, как во внешнем: элементы красятся по порядковому номеру, чётные в один цвет, нечётные в другой

но для начала у самого списка .town-advantages-list должен быть белый фоновый цвет, а его нет. фон для списка у тебя нигде не указан, поэтому он и не красится в белый, как ты ожидала
у .housing.town-advantages-item и .souvenirstown-advantages-item тоже незаметно фонового цвета, потому что rgba(131, 179, 211, 0.12) это очень прозрачный цвет 
![HTML Academy: Седона 2022-06-03 00-23-51](https://user-images.githubusercontent.com/11165551/171740844-323176c9-31d9-4153-8b04-c1b1b75edd2b.jpg)
то есть пункты красятся, этого просто не видно. видно будет, если под пунктами будет белая подложка, то есть, если ты сам список покрасишь в белый

OlgaNikl committed:
исправлены недочеты

изменен фон и основной шрифт у body;  изменены классы и переназначены селекторы у списков;

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,297 @@
+  background-color: #F2F2F2;
+}
+
+.main-conteiner {
```
опечатка, container

talequale commented on styles/styles.css:
```diff
@@ -0,0 +1,297 @@
+}
+
+.main-conteiner {
+  background-color: #ffffff;
```
помимо фонового цвета можно сразу эти стили добавить 
![HTML Academy: Седона 2022-06-04 21-22-04](https://user-images.githubusercontent.com/11165551/172020652-e88707dd-37c2-426d-9545-573c8b932a70.jpg)
